.github/actions/policy-check-image/action.yml

```diff
@@ -5,20 +5,44 @@ inputs:
 runs:
   using: composite
   steps:
-    # Note: this matches what is currently used for policy-controller
-    - uses: actions/setup-go@d0a58c1c4d2b25278816e339b944508c875f3613
-      with:
-        go-version: '1.18'
-        check-latest: true
-    - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
-      with:
-        repository: sigstore/policy-controller
-        path: ${{github.workspace}}/policy-controller
-    - name: Build the policy-tester CLI
+    - name: Build and image with policy-tester and crane
       shell: bash
       run: |
-        (cd policy-controller/ && make policy-tester && mv policy-tester /tmp/policy-tester)
-    - uses: imjasonh/setup-crane@v0.1
+        trap "rm -rf custom-image.tar custom-image.apko.yaml" EXIT
+
+        cat >./custom-image.apko.yaml <<EOL
+        contents:
+          packages:
+            - glibc # TODO: fix policy-controller-tester package to not need this or automatically require it
```

Comment:
TODO should be dropped when new policy-controller-tester is built: https://github.com/wolfi-dev/os/pull/530

Reply:
hm.. doesnt seem to fix the issue